playbooks/roles/caasp4/templates/skuba-terraform-libvirt.tfvars.j2

```diff
@@ -13,31 +13,30 @@ masters = {{ master_count }}
 workers = {{ worker_count }}
 
 repositories = {
-  caasp_devel = "{{ ibs_mirror_uri }}/Devel:/CaaSP:/4.0/SLE_15_SP1/"
-  suse_ca = "{{ ibs_mirror_uri }}/SUSE:/CA/SLE_15_SP1/"
-  sle_server_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Product-SLES/15-SP1/x86_64/product/"
-  basesystem_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/"
-  containers_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Module-Containers/15-SP1/x86_64/product/"
-  serverapps_pool = "{{ rmt_mirror_uri }}/SUSE/Products/SLE-Module-Server-Applications/15-SP1/x86_64/product/"
-  sle_server_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Product-SLES/15-SP1/x86_64/update/"
-  basesystem_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/"
-  containers_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Module-Containers/15-SP1/x86_64/update/"
-  serverapps_updates = "{{ rmt_mirror_uri }}/SUSE/Updates/SLE-Module-Server-Applications/15-SP1/x86_64/update/"
+  caasp_pool = "{{ ibs_mirror }}/SUSE/Products/SUSE-CAASP/4.0/x86_64/product/"
+  sle_server_pool = "{{ ibs_mirror }}/SUSE/Products/SLE-Product-SLES/15-SP1/x86_64/product/"
```

Comment:
Could you update the locations in `skuba-terraform-openstack.tfvars.j2` as well? (could be in separate PR if you want).